python/lsst/afw/cameraGeom/cameraFactory.py

```diff
@@ -223,14 +187,15 @@ def makeCameraFromPath(cameraConfig, ampInfoPath, shortNameFunc,
 
 def makeCameraFromAmpLists(cameraConfig, ampListDict,
                            pupilFactoryClass=PupilFactory):
-    """Construct a Camera instance from a dictionary of detector name: AmpInfoCatalog
+    """Construct a Camera instance from a dictionary of detector name: list of
+    Amplifier.Builder
 
     Parameters
     ----------
     cameraConfig : `CameraConfig`
         Config describing camera and its detectors.
     ampListDict : `dict`
-        A dictionary of detector name: AmpInfoCatalog
+        A dictionary of detector name: list of Amplifier.Builder
```

Comment:
I actually meant the style of the description (no colon notation) rather than the type; `dict` is valid for key or value types that are either open-ended or too complicated to describe. :wink: